ui/client/local-types/reactable/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Why we have another package.json in project? It is kind of new module?

Reply:
https://github.com/TouK/nussknacker/pull/1054/files#r456340877